contributing-guides/style-guide.zh.md

```diff
@@ -0,0 +1,90 @@
+# 格式指导
+
+当你在为 `tldr` 贡献时，请遵守下面的格式规范。
+
+请注意，下面的规范仅适用于中文翻译的 `tldr` 页面。
+
+## 排版
+
+首先，你的页面应该看起来像这样：
+
+```
+# 命令名称
+
+> 短小精悍的描述。
+> 描述最好只有一行；当然，如果需要，也可以是两行。
+> 更多信息：<https://example.com>。
```

Comment:
I believe the end of the line should be `.` not the `。`

I remember there was some discussion before, but we still not reach an agreement.

Reply:
As the description said, I _personally_ use `.`, but figured that it's simpler to just follow the guideline stated above.

Reply:
The Chinese period looks weird even to me after the Latin characters.  Maybe just "always use the Latin period at the end of a sentence when the last character of the sentence is Latin"?